tests/ui/html/ui-storyEditor-manual-Tests.html

```diff
@@ -0,0 +1,202 @@
+<html>
```

Comment:
Could you add a documentation to describe available manual tests, their goals and how to access, etc? Thanks.